src/main/java/frc/robot/vision/limelights/LimeLightConstants.java

```diff
@@ -0,0 +1,36 @@
+package frc.robot.vision.limelights;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class LimeLightConstants {
+
+	public static final String ESTIMATION_LOGPATH_PREFIX = "Estimation";
+
+	public static final int LIMELIGHT_ENTRY_ARRAY_LENGTH = 7;
+
+
+	public static final LimelightFiltererConfig DEFAULT_CONFIG = new LimelightFiltererConfig(
+		"VisionObservationFiltered/",
+		"LimelightsHardware/",
+		new String[] {"limelight-front", "limelight-back"}
+	);
+
+	public static final double ROTATION_NORM_TOLERANCE = 0.2; // ! shall be calibrated
```

Comment:
A sneaky way to not write "TODO" 🙃
But fine since this is calibration

Reply:
I will just remove it since I already calibrated it🚄